app/services/ems_dashboard_service.rb

```diff
@@ -1,6 +1,9 @@
 class EmsDashboardService < DashboardService
   attr_reader :ems
   include Mixins::CheckedIdMixin
+  include TextualSummaryHelper
+  include ActionView::Helpers::DateHelper
+  include TextualMixins::TextualRefreshStatus
```

Comment:
Also for consistency, let's put all of the includes in the same order across the various dashboard files.

Reply:
i am not really what that mean, but rearranged the order, let me know if i need t make any more changes.

Reply:
Oh I was saying just alphabetize unless the order matters...when the order matters, we should just be consistent across all of the services.